### ISSUE TITLE
Question on Batch Normalization Layer

### ISSUE THREAD

**Issue by hustsxy** [boluoweifenda/WAGE#13]
Thanks for the great work! I noticed that in the paper you did some comparison experiments with BN layer. I tried to insert BN layer into the current network but it actually makes the accuracy much worse. (I removed the weight scaling factor part).  Is there anything I missed? Could you please give any suggestions on this issue? Thanks!

**Comment by boluoweifenda** [comment]
@hustsxy 
BN layers run in the floating-point mode, so they are quite different from the quantized weights and should be addressed more carefully.
For example, without \beta and \gamma, the output of BN have a distribution with 1 std, which only contains part of values in the range [-1, +1]. Then the original quantization range would not include most of the values, let alone the scaling effect of \gamma.
Besides, the learning rate of weight in the paper has been modified, but it is not suitable for floating-point representation \beta and \gamma.
